templates/blog/post_list.html

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html' %}
+
+{% block title %}
+  <title>Blog Posts</title>
+{% endblock %}
+
+{% block content %}
+  <h2>Posts</h2>
+  <div>
+    <ul>
+      {% for post in post_list %}
+        <li>
+          <h3><a href="{% url 'blog:post-detail' pk=post.id %}">{{ post.title }}</a></h3>
+          <div> Author: {{ post.owner }}</div>
+          <div>{{ post.created_time }}</div>
```

Comment:
The `created_time` is being displayed without any formatting. This might result in a not very user-friendly timestamp. Consider using Django's date filter to format the date in a more readable way. For example: `{{ post.created_time|date:'F d, Y' }}`.